package.json

```diff
@@ -22,7 +22,7 @@
         "resourceDetailsTabs": [
           {
             "resourceTypes": [
-              "service"
+              "service", "server"
```

Comment:
Don't forget to run `npm run format` and `npm run lint` before a PR. They're not automated in this repo (because we don't want to commit customers to our github workflows).

You can set up VSCode to always format on save to help this too.

Reply:
run format went smoothly but `npm run lint` threw this error @orvisevans 

```
> cui-applet@0.1.0 lint
> eslint --ext .js,.vue --ignore-path .gitignore --fix src


Oops! Something went wrong! :(

ESLint: 8.43.0

Error [ERR_REQUIRE_ESM]: require() of ES Module /Users/atorres/dev/cb-applet/.eslintrc.js from /Users/atorres/dev/cb-applet/node_modules/@eslint/eslintrc/dist/eslintrc.cjs not supported.
.eslintrc.js is treated as an ES module file as it is a .js file whose nearest parent package.json contains "type": "module" which declares all .js files in that package scope as ES modules.
Instead rename .eslintrc.js to end in .cjs, change the requiring code to use dynamic import() which is available in all CommonJS modules, or change "type": "module" to "type": "commonjs" in /Users/atorres/dev/cb-applet/package.json to treat all .js files as CommonJS (using .mjs for all ES modules instead).

    at module.exports [as default] (/Users/atorres/dev/cb-applet/node_modules/import-fresh/index.js:32:59)
    at loadJSConfigFile (/Users/atorres/dev/cb-applet/node_modules/@eslint/eslintrc/dist/eslintrc.cjs:2562:47)
    at loadConfigFile (/Users/atorres/dev/cb-applet/node_modules/@eslint/eslintrc/dist/eslintrc.cjs:2646:20)
    at ConfigArrayFactory.loadInDirectory (/Users/atorres/dev/cb-applet/node_modules/@eslint/eslintrc/dist/eslintrc.cjs:2856:34)
    at CascadingConfigArrayFactory._loadConfigInAncestors (/Users/atorres/dev/cb-applet/node_modules/@eslint/eslintrc/dist/eslintrc.cjs:3848:46)
    at CascadingConfigArrayFactory._loadConfigInAncestors (/Users/atorres/dev/cb-applet/node_modules/@eslint/eslintrc/dist/eslintrc.cjs:3867:20)
    at CascadingConfigArrayFactory.getConfigArrayForFile (/Users/atorres/dev/cb-applet/node_modules/@eslint/eslintrc/dist/eslintrc.cjs:3769:18)
    at FileEnumerator._iterateFilesRecursive (/Users/atorres/dev/cb-applet/node_modules/eslint/lib/cli-engine/file-enumerator.js:450:49)
    at _iterateFilesRecursive.next (<anonymous>)
    at FileEnumerator.iterateFiles (/Users/atorres/dev/cb-applet/node_modules/eslint/lib/cli-engine/file-enumerator.js:299:49)
    at iterateFiles.next (<anonymous>)
    at CLIEngine.executeOnFiles (/Users/atorres/dev/cb-applet/node_modules/eslint/lib/cli-engine/cli-engine.js:788:48)
    at ESLint.lintFiles (/Users/atorres/dev/cb-applet/node_modules/eslint/lib/eslint/eslint.js:551:23)
    at Object.execute (/Users/atorres/dev/cb-applet/node_modules/eslint/lib/cli.js:391:36)
    at async main (/Users/atorres/dev/cb-applet/node_modules/eslint/bin/eslint.js:135:24)
```

Reply:
Renaming the file to .eslintrc.cjs (as suggested by the error) worked for me.

Reply:
Doing so runs and gives me back 4 lint errors:

```bash
/Users/orvisevans/src/cb-applet/src/ACounter.vue
  10:3  error  The template root requires exactly one element  vue/no-multiple-template-root

/Users/orvisevans/src/cb-applet/src/MainView.vue
  26:3  error  The template root requires exactly one element  vue/no-multiple-template-root
  53:7  error  'props' is assigned a value but never used      no-unused-vars

/Users/orvisevans/src/cb-applet/src/ResourceDetailView.vue
  18:7  error  'props' is assigned a value but never used  no-unused-vars

✖ 4 problems (4 errors, 0 warnings)
```

Reply:
I'll do this change and commit it.